tests/Compilation/positive/test060.juvix

```diff
@@ -17,7 +17,8 @@ type Pair (A B : Type) :=
     };
 
 mf : Pair (Pair Bool (List Nat)) (List Nat) → Bool
-  | mkPair@{fst := mkPair@{fst; snd := nil}; snd := zero :: _} := fst
```

Comment:
These changes are not related to the topic of this PR. Undo this.

Reply:
Yes, I will. It's the work of the formatter.

Reply:
done